client/components/voip/composer/NotAvailableOnCall/NotAvailable.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { Story } from '@storybook/react';
+import React from 'react';
+
+import NotAvailableContent from './NotAvailableContent';
+import NotAvailableContentWrapper from './NotAvailableContentWrapper';
+
+export const Default: Story = () => (
+	<NotAvailableContentWrapper>
+		<NotAvailableContent text='Messages are not available on phone calls' icon='message' />
+	</NotAvailableContentWrapper>
```

Comment:
Why not use `NotAvailableOnCall` directly here?

Reply:
this component have useTranslation on it...